### ISSUE TITLE
Feature-Request: Best Practical RT/RTIR connector for tracker notifications

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#36]
It would be very helpful for a lot of CERTs to provide a RT/RTIR connector in order to be able, for example, to send out tracker notifications via the ticket system (incident reports, investigations, tickets) to constituents directly with a custom description in a definable body and subject. As a basis, for example, the already available python API from https://github.com/CZ-NIC/python-rt could be a starting point.

**Comment by adulau** [comment]
Good idea. For the time being, the email notification could do the job via a dedicated email to a specific RTIR queue ;-)
